lib/packwerk/cli.rb

```diff
@@ -57,7 +57,11 @@ def execute_command(args)
       when "detect-stale-violations"
         output_result(parse_run(args).detect_stale_violations)
       when "update-deprecations"
-        output_result(parse_run(args).update_deprecations)
+        output_result(parse_run(args).update_todo)
```

Comment:
Need to raise a deprecation warning here.

Reply:
Added a warning.